### ISSUE TITLE
If the player steps on the half brick, it will be judged as FlyHacker.

### ISSUE THREAD

**Issue by KaguraMatsuri** [xaw3ep/NoCheatPlus#56]
3.17-SNAPSHOT-b1153-CustomForkn

If the player steps on the half brick, it will be judged as FlyHacker.

### Any possible config options changed or plugins that may cause interference?
Yes

**Comment by MarkElf** [comment]
Unable to reproduce, provide a debug log (/ncp debug player {name}) and the output of /ncp version.

**Comment by KaguraMatsuri** [comment]
[09:44:11 INFO]: ---- Version information ----
[09:44:11 INFO]: #### Server ####
[09:44:11 INFO]: git-PaperSpigot-f6fba00-9935adc ~MC: 1.8.8~
[09:44:11 INFO]:&nbsp; &nbsp;detected: 1.8.8
[09:44:11 INFO]: #### NoCheatPlus ####
[09:44:11 INFO]: Plugin: 3.17-SNAPSHOT-b1153-CustomFork
[09:44:11 INFO]: MCAccess: 1.6.1-1.14.4|? / Bukkit-API
[09:44:11 INFO]: Features:
[09:44:11 INFO]:&nbsp; &nbsp;blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | MCAccessBukkit
[09:44:11 INFO]:&nbsp; &nbsp;checks: FastConsume | Gutenberg | AttackFrequency | FlyingFrequency | KeepAliveFrequency
[09:44:11 INFO]:&nbsp; &nbsp;defaults: pvpKnockBackVelocity
[09:44:11 INFO]:&nbsp; &nbsp;packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance | WrongTurnAdapter | CatchAllAdapter | NoSlow | Fight
[09:44:11 INFO]: Hooks: AllViolations~NCP~ 1.0 | ViolationFrequency~NCP~ 1.0
[09:44:11 INFO]: #### Related Plugins ####
[09:44:11 INFO]: ProtocolLib v4.5.0




[09:43:26 INFO]: UUID of player SaionjiSekai is 41062b51-1942-4276-8579-38e1c46c15e7
[09:43:32 INFO]: SaionjiSekai[/127.0.0.1:7881] logged in with entity id 95 at ([world]-0.504075240283177, 109.0, 101.48870281052326)
[09:43:32 INFO]: [NoCheatPlus] PacketPlayOutPosition: Teleport confirm id not available, field mismatch: 0
[09:43:57 INFO]: [NC+] SaionjiSekai failed KeepAliveFrequency: is spamming keep-alive packets to the server (VL12)
&gt;ncp version
[09:44:11 INFO]: ---- Version information ----
[09:44:11 INFO]: #### Server ####
[09:44:11 INFO]: git-PaperSpigot-f6fba00-9935adc ~MC: 1.8.8~
[09:44:11 INFO]:&nbsp; &nbsp;detected: 1.8.8
[09:44:11 INFO]: #### NoCheatPlus ####
[09:44:11 INFO]: Plugin: 3.17-SNAPSHOT-b1153-CustomFork
[09:44:11 INFO]: MCAccess: 1.6.1-1.14.4|? / Bukkit-API
[09:44:11 INFO]: Features:
[09:44:11 INFO]:&nbsp; &nbsp;blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | MCAccessBukkit
[09:44:11 INFO]:&nbsp; &nbsp;checks: FastConsume | Gutenberg | AttackFrequency | FlyingFrequency | KeepAliveFrequency
[09:44:11 INFO]:&nbsp; &nbsp;defaults: pvpKnockBackVelocity
[09:44:11 INFO]:&nbsp; &nbsp;packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance | WrongTurnAdapter | CatchAllAdapter | NoSlow | Fight
[09:44:11 INFO]: Hooks: AllViolations~NCP~ 1.0 | ViolationFrequency~NCP~ 1.0
[09:44:11 INFO]: #### Related Plugins ####
[09:44:11 INFO]: ProtocolLib v4.5.0
[09:45:07 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to move unexpectedly (VL187)
[09:45:12 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to move unexpectedly (VL664)
[09:45:17 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to move unexpectedly (VL194)
[09:45:20 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to move unexpectedly (VL347)
[09:45:31 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to perform an illegal move (Subchecks: hover) (VL1169)
&gt;ncp debug player SaionjiSekai
[09:45:35 INFO]: Set debug = YES for player SaionjiSekai for checks: ALL
[09:45:43 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to perform an illegal move (Subchecks: dirty+bunnyenv+bunnyenv+hspeed) (VL1652)
&gt;ncp debug player SaionjiSekai yes
[09:45:55 INFO]: Set debug = YES for player SaionjiSekai for checks: ALL
[09:45:55 INFO]: Not online: yes
[09:46:03 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to move unexpectedly (VL236)
[09:46:06 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to move unexpectedly (VL301)
[09:46:15 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to move unexpectedly (VL359)
[09:46:23 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to move unexpectedly (VL779)
[09:46:23 INFO]: [NC+] SaionjiSekai failed SurvivalFly: tried to perform an illegal move (Subchecks: bunnyenv+bunnyenv+hspeed+data_reset+step+vdistrel) (VL911)
&gt;








------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"MarkElf"<notifications@github.com&gt;;
发送时间:&nbsp;2020年5月22日(星期五) 凌晨1:31
收件人:&nbsp;"xaw3ep/NoCheatPlus"<NoCheatPlus@noreply.github.com&gt;;
抄送:&nbsp;"神楽祭"<1283962190@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [xaw3ep/NoCheatPlus] If the player steps on the half brick, it will be judged as FlyHacker. (#56)





 
Unable to reproduce, provide a debug log (/ncp debug player {name}) and the output of /ncp version.

**Comment by MarkElf** [comment]
Very outdated version.

**Comment by KaguraMatsuri** [comment]
Yes, because 3 months have passed before you reply to my email, and the latest NCP did not fix this vulnerability.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xaw3ep/NoCheatPlus"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年8月1日(星期六) 中午12:17
收件人:&nbsp;"xaw3ep/NoCheatPlus"<NoCheatPlus@noreply.github.com&gt;;
抄送:&nbsp;"神樂 祭"<1283962190@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [xaw3ep/NoCheatPlus] If the player steps on the half brick, it will be judged as FlyHacker. (#56)





 
Very outdated version.
